### PR TITLE
Feat/tooltip line style

### LINKED ISSUE #338: [Feature] tooltip support configure single content line style

### PR DESCRIPTION
**Pull request by zamhown**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
close #338 

### 🐞 Bugserver case id
64dc73f1ae69d5156f811c0a

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

**Comment by kkxxkk2019** [comment]
BUGSERVER 有几个 badcase 看一下

image

image

File: common/changes/@visactor/vchart/feat-tooltip-line-style_2023-08-02-13-16.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@visactor/vchart",
+      "comment": "feat: tooltip support configure single content line style, related #338",
```
Comment (kkxxkk2019) [inline common/changes/@visactor/vchart/feat-tooltip-line-style_2023-08-02-13-16.json:5]: 这个 bugserver 也添加下 case 吧

**Review by kkxxkk2019 — APPROVED** [review]

**Comment by kkxxkk2019** [comment]
简单说明下用法？

File: common/changes/@visactor/vchart/feat-tooltip-line-style_2023-08-02-13-16.json (hunk 1 above)
Comment (zamhown) [reply common/changes/@visactor/vchart/feat-tooltip-line-style_2023-08-02-13-16.json:5]: 加过了 https://bugserver.cn.goofy.app/case?product=chartspace4&fileid=64dc73f1ae69d5156f811c0a&interactionid=64dc742dae69d5156f811c0b

**Comment by zamhown** [comment]
> 简单说明下用法？

每行和 key、value 同级的地方添加了 keyStyle 和 valueStyle，支持回调，且支持在 updateContent 回调中单独修改 data join 后的某行的样式

**Pull request merged**